protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -122,6 +122,31 @@ definitions:
           - SOURCE_RETRIEVAL_ERROR
           # Errors casting to appropriate type
           - DESTINATION_TYPECAST_ERROR
+  AirbyteFileTransferRecordMessage:
```

Comment:
Should we rename this to `AirbyteFileTransferMessage` It is not really a record here.

Reply:
Renamed/